policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml

```diff
@@ -0,0 +1,28 @@
+description: Arkansas accounts for these exemptions when calculating the state adjusted gross income.
+values:
+  2021-01-01:
+    - military_retirement_pay # Item 1
+    # Item 2&3 Life insurance policy
+    - child_support_received # Item 4
+    # Item 5 Gifts, inheritances, bequests, or devises
+    # Item 6 Scholarships, fellowships, and grants 
+    - us_govt_interest # Item 7
+    - workers_compensation # Item 8
```

Comment:
need to check what VA benefits - I think its veteran related

Reply:
Found "veterans_benefits". Does that fit?

> need to check what VA benefits - I think its veteran related